spdx-utils/src/main/kotlin/model/SpdxDocument.kt

```diff
@@ -138,5 +138,11 @@ data class SpdxDocument(
         require(dataLicense.isNotBlank()) { "The data license must not be blank." }
 
         require(documentNamespace.isNotBlank()) { "The document namespace must not be blank." }
+
+        require((relationships.isNotEmpty() && relationships.any { it ->
```

Comment:
I believe `relationships.isNotEmpty()` in implied by `relationships.any {}` and can be omitted. The `it ->` can also be omitted (naming the lambda parameter `it` is the default).